packages/sdk/react-client/src/echo/useSchema.ts

```diff
@@ -2,21 +2,19 @@
 // Copyright 2025 DXOS.org
 //
 
-import type * as Schema from 'effect/Schema';
 import { useMemo, useSyncExternalStore } from 'react';
 
 import { type Client } from '@dxos/client';
-import { type Space } from '@dxos/client/echo';
-import { type BaseSchema } from '@dxos/echo/internal';
+import { type Space, type Type } from '@dxos/client/echo';
 
 /**
  * Subscribe to and retrieve schema changes from a space's schema registry.
  */
-export const useSchema = (
+export const useSchema = <T extends Type.Obj.Any = Type.Obj.Any>(
```

Comment:
Should be `Type.Entity.Any` (which might also help w/ the cast below since there's some type inference issues w/ the object-specific schema type currently)